imi/src/main/java/org/motechproject/nms/imi/domain/CallDetailRecord.java

```diff
@@ -9,7 +9,7 @@
  * See NMS API - 4.4.3 CDR Detail File Format
  */
 @Entity(tableName = "nms_imi_cdrs")
-@Index(members = { "requestId" })
+@Index(name = "requestId_callId_composite_idx", members = { "requestId", "callId" })
```

Comment:
Kosh: "requestIId is shared for all CDR tries on a given day. requestId+callId is what we need to de-dupe with"